authors/apps/authentication/urls.py

```diff
@@ -1,12 +1,17 @@
 from django.conf.urls import url
 
 from .views import (
-    LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView
+    LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView,
+    ForgotPasswordAPIview, ResetPasswordAPIView
 )
 app_name = 'authentication'
 
+
 urlpatterns = [
-    url(r'^user/?$', UserRetrieveUpdateAPIView.as_view()),
```

Comment:
Any reason we are getting rid of this endpoint

Reply:
Yeah I think I will remove that

Reply:
@jonathanmusila the reason

Reply:
I think I got you now, no reason to get rid of it, I think I removed that by mistake, will correct that